scripts/upgrade

```diff
@@ -20,6 +20,7 @@ domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
 final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 db_name=$(ynh_app_setting_get --app=$app --key=db_name)
+db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
```

Comment:
If you want this variable here (?) you should remove the line https://github.com/YunoHost-Apps/ttrss_ynh/blob/86c1f35ab8492dec124869365ff44bec50e4173e/scripts/upgrade#L143

Reply:
Do you mean that I don't need the db_pwd variable and I should remove it alltogether?

Reply:
No, I mean we should only keep one of these lines